### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by daphneherlambda**
This PR updates the proyect readme to contain the docker container section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
  * [honggfuzz](https://github.com/google/honggfuzz)
  * [cargofuzz](https://github.com/rust-fuzz/cargo-fuzz)
 
+Fuzzers structure 
+---
+
+In this repo the fuzzers have the following structure:
+
+### if the fuzzer uses honggfuzz
```
Comment (Loco1982) [inline README.md]:
```suggestion
### Honggfuzz' fuzzer
```

File: README.md [hunk 2]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
...
+---
+
+In this repo the fuzzers have the following structure:
+
+### if the fuzzer uses honggfuzz 
+
+* The  ***src*** folder contains the file main with the fuzzer 
+* The ***hfuzz_workspace*** folder contains a REPORT file with the report of the crashes the fuzzer found, this folder is updated each time the fuzzer finds a new crash and has the information about the crash along with the name of the input that triggered it. Also contains the input files that triggers the errors, the file names starts with SIGABRT
+
+### if the fuzzer uses cargofuzz
```
Comment (Loco1982) [inline README.md]:
```suggestion
### Cargofuzz' fuzzer
```

File: README.md [hunk 3]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
...
 
+Fuzzers structure 
+---
+
+In this repo the fuzzers have the following structure:
+
+### if the fuzzer uses honggfuzz 
+
+* The  ***src*** folder contains the file main with the fuzzer 
+* The ***hfuzz_workspace*** folder contains a REPORT file with the report of the crashes the fuzzer found, this folder is updated each time the fuzzer finds a new crash and has the information about the crash along with the name of the input that triggered it. Also contains the input files that triggers the errors, the file names starts with SIGABRT
```
Comment (Loco1982) [inline README.md]:
```suggestion
* The ***hfuzz_workspace*** folder contains a REPORT file with the report of the crashes the fuzzer found, this folder is updated each time the fuzzer finds a new crash and has the information about the crash along with the name of the input that triggered it. Also contains the input files that trigger the errors, the file names start with SIGABRT
```

File: README.md [hunk 4]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
...
+
+### if the fuzzer uses cargofuzz 
+
+* the ***fuzz_targets*** contains the files with the fuzzers
+
+
```
Comment (Loco1982) [inline README.md]:
```suggestion
* the ***fuzz_targets*** contains the files with the fuzzers

```

It has one too many new lines

File: README.md [hunk 5]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
...
 ## User Guide
 
+### 1. create the docker container 
+* Use the command  `docker build -t ready-to-fuzz-image` inside this proyect to create a docker image named **ready-to-fuzz-image** from the dockerfile in the proyect.
```
Comment (Loco1982) [inline README.md]:
```suggestion

* Use the command  `docker build -t ready-to-fuzz-image` inside this project to create a docker image named **ready-to-fuzz-image** from the dockerfile in the project.
```

File: README.md [hunk 6]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
...
+* run the command `docker run -it --rm --cap-add=SYS_PTRACE --security-opt seccomp=unconfined ready-to-fuzz-image bash` to run the container from the image previusly created.
```
Comment (Loco1982) [inline README.md]:
```suggestion
* run the command `docker run -it --rm --cap-add=SYS_PTRACE --security-opt seccomp=unconfined ready-to-fuzz-image bash` to run the container from the image previously created.
```

**Review by Loco1982 — CHANGES_REQUESTED** [review]
Made a few comments, looks good!

File: README.md [hunk 7]
```diff
@@ -18,16 +14,37 @@ To see the usage of this libraries check their documentation:
...
+
+### 2. Run a fuzzer 
+
 Depending on the fuzzer use the following commands:
 
 ### If the Fuzzer Uses Honggfuzz
 
 * to run it with total random data use:
```
Comment (Loco1982) [inline README.md]:
```suggestion
* to run it with totally random data use: 
```

File: README.md (hunk 1 above)
Comment (daphneherlambda) [reply README.md]: done!

File: README.md (hunk 2 above)
Comment (daphneherlambda) [reply README.md]: done!

File: README.md (hunk 3 above)
Comment (daphneherlambda) [reply README.md]: done!

File: README.md (hunk 4 above)
Comment (daphneherlambda) [reply README.md]: done!

File: README.md (hunk 5 above)
Comment (daphneherlambda) [reply README.md]: done!

File: README.md (hunk 6 above)
Comment (daphneherlambda) [reply README.md]: done!

File: README.md (hunk 7 above)
Comment (daphneherlambda) [reply README.md]: ups typo, fixed!

File: README.md (hunk 5 above)
Comment (Loco1982) [reply README.md]: It should say 'project' not 'proyect'

**Review by Loco1982 — CHANGES_REQUESTED** [review]
There's a small issue left for me to approve this one

**Review by Loco1982 — APPROVED** [review]
LGTM

**Pull request merged**